lib/sales_analyst.rb

```diff
@@ -34,10 +36,21 @@ def average_item_price_for_merchant(merchant_id)
   end
 
   def average_average_price_per_merchant
-    total = @sales_engine.merchants.all.sum do |merchant|
+    total = merchants.all.sum do |merchant|
       average_item_price_for_merchant(merchant.id)
     end
-    (total / @sales_engine.merchants.all.count).round(2)
+    (total / merchants.all.count).round(2)
   end
 
+  def merchants
+    @sales_engine.merchants
+  end
+
+  def items
+    @sales_engine.items
+  end
+
```

Comment:
I find it interesting that merchants and items are here as well. I'd like to discuss that later but all and all this looks good. Thanks for taking over the stdev stuff. I deffo need to improve my math skills and be a little more confident in my abilities to tackle them. I'm sorry you had to do this method but I'm happy with it. You were saying it looked horrible and I disagree. I think it looks clean and just like it should and we can re-factor later if we can push out it1 - 4 or 5.